### PR TITLE
Fercobo msft dataset updates

### PR DESCRIPTION
**Pull request by fercobo-msft**
Adding Teams datasets.

### CODE REVIEW

**Comment by David1997sb** [comment]
This PR has been open since Jan and I have not received the pending datasets (SharepointSitesDataset_v0_Preview, and TeamsStandardChannelMessages) please tag me when this information is available. I will be closing the workitem associated with it [Substrate Docs Stories Board - Boards (visualstudio.com)](https://o365exchange.visualstudio.com/DefaultCollection/O365%20Sandbox/_boards/board/t/Substrate%20Docs/Stories/?workitem=2582223) and I can reopen it when the information of the datasets is shared with me.
@NikCharlebois

**Review by rimisra2 — COMMENTED** [review]
Hey David, please make sure that the comma after pipelines is removed in the data-connect-pam file. Thanks!

**Review by rimisra2 — APPROVED** [review]

File: concepts/data-connect-datasets.md [hunk 1]
```diff
@@ -14,26 +14,71 @@ Microsoft Graph Data Connect supports a variety of datasets, data regions, and s
...
+| BasicDataset_v0.TeamChannelDetails_v0 | Contains a Teams Channel details. | <ul><li> [Sample](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/sampledatasets/TeamsChannelDetails_v0.json)</li><li>[Schema](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/datasetschemas/TeamsChannelDetails_v0.md)</li></ul> | [Paid](https://azure.microsoft.com/pricing/details/graph-data-connect/) |
+| BasicDataset_v0.TeamStandardChannelMessages_v0 | Contains a Teams standard channel messages details. | <ul><li> [Sample](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/sampledatasets/BasicDataSet_v0.TeamsStandardChannelMessages_v0.json)</li><li>[Schema](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/datasetschemas/TeamsStandardChannelMessages_v0.md)</li></ul> | [Paid](https://azure.microsoft.com/pricing/details/graph-data-connect/) |
```
Comment (Danielabom) [inline concepts/data-connect-datasets.md:52]:
```suggestion
| BasicDataset_v0.TeamStandardChannelMessages_v0 | Contains a Teams standard channel message details. | <ul><li> [Sample](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/sampledatasets/BasicDataSet_v0.TeamsStandardChannelMessages_v0.json)</li><li>[Schema](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/datasetschemas/TeamsStandardChannelMessages_v0.md)</li></ul> | [Paid](https://azure.microsoft.com/pricing/details/graph-data-connect/) |
```

File: concepts/data-connect-datasets.md [hunk 2]
```diff
@@ -14,26 +14,71 @@ Microsoft Graph Data Connect supports a variety of datasets, data regions, and s
...
+| BasicDataSet_v0.TeamsChannelDetails_v0  | Contains details about Channels in a team.  | <ul><li> [Sample](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/sampledatasets/BasicDataSet_v0.TeamsChannelDetails_v0.json)</li><li>[Schema](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/datasetschemas/BasicDataSet_v0.TeamsChannelDetails_v0%20.md)</li></ul> | [Paid](https://azure.microsoft.com/pricing/details/graph-data-connect/) |
+| BasicDataSet_v0.TeamsStandardChannelMessages_v0  | Contains channel posts, and messages from Standard Channels in Teams.    | <ul><li> [Sample](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/sampledatasets/BasicDataSet_v0.TeamsStandardChannelMessages_v0.json)</li><li>[Schema](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/datasetschemas/BasicDataSet_v0.TeamsStandardChannelMessages_v0%20.md)</li></ul> | [Paid](https://azure.microsoft.com/pricing/details/graph-data-connect/) |
```
Comment (Danielabom) [inline concepts/data-connect-datasets.md:55]:
```suggestion
| BasicDataSet_v0.TeamsStandardChannelMessages_v0  | Contains channel posts and messages from Standard Channels in Teams.    | <ul><li> [Sample](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/sampledatasets/BasicDataSet_v0.TeamsStandardChannelMessages_v0.json)</li><li>[Schema](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/datasetschemas/BasicDataSet_v0.TeamsStandardChannelMessages_v0%20.md)</li></ul> | [Paid](https://azure.microsoft.com/pricing/details/graph-data-connect/) |
```

File: concepts/data-connect-datasets.md [hunk 3]
```diff
@@ -14,26 +14,71 @@ Microsoft Graph Data Connect supports a variety of datasets, data regions, and s
...
+| BasicDataSet_v0.GroupMembers_v0 | Contains a Teams Chat Group member details. | <ul><li> [Sample](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/sampledatasets/GroupMembers_v0.json)</li><li>[Schema](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/datasetschemas/GroupMembers_v0.md)</li></ul> | [Free](https://azure.microsoft.com/pricing/details/graph-data-connect/) |
+| BasicDataSet_v0.GroupOwners_v0 | Contains the Teams Chat Group owners details. | <ul><li> [Sample](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/sampledatasets/GroupOwners_v0.json)</li><li>[Schema](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/datasetschemas/GroupOwners_v0.md)</li></ul> | [Free](https://azure.microsoft.com/pricing/details/graph-data-connect/) |
```
Comment (Danielabom) [inline concepts/data-connect-datasets.md:64]:
```suggestion
| BasicDataSet_v0.GroupOwners_v0 | Contains the Teams Chat Group owners' details. | <ul><li> [Sample](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/sampledatasets/GroupOwners_v0.json)</li><li>[Schema](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/datasetschemas/GroupOwners_v0.md)</li></ul> | [Free](https://azure.microsoft.com/pricing/details/graph-data-connect/) |
```

File: concepts/data-connect-datasets.md [hunk 4]
```diff
@@ -14,26 +14,71 @@ Microsoft Graph Data Connect supports a variety of datasets, data regions, and s
...
+
+### OneDrive and SharePoint Online
+
+Please consider that these datasets are available after 48 hours (about 2 days). For instance, you can query data for 07/01 starting on 07/03.
```
Comment (Danielabom) [inline concepts/data-connect-datasets.md:68]:
```suggestion
Please consider that these datasets are available after 48 hours (about two days). For instance, you can query data for 07/01 starting on 07/03.  
```

File: concepts/data-connect-datasets.md [hunk 5]
```diff
@@ -14,26 +14,71 @@ Microsoft Graph Data Connect supports a variety of datasets, data regions, and s
...
+| BasicDataSet_v0.GroupMembers_v0 | Contains a Teams Chat Group member details. | <ul><li> [Sample](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/sampledatasets/GroupMembers_v0.json)</li><li>[Schema](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/datasetschemas/GroupMembers_v0.md)</li></ul> | [Free](https://azure.microsoft.com/pricing/details/graph-data-connect/) |
+| BasicDataSet_v0.GroupOwners_v0 | Contains the Teams Chat Group owners details. | <ul><li> [Sample](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/sampledatasets/GroupOwners_v0.json)</li><li>[Schema](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/datasetschemas/GroupOwners_v0.md)</li></ul> | [Free](https://azure.microsoft.com/pricing/details/graph-data-connect/) |
+
+### OneDrive and SharePoint Online
+
+Please consider that these datasets are available after 48 hours (about 2 days). For instance, you can query data for 07/01 starting on 07/03.  
+
+This data is available for only the past 21 days (about 3 weeks). For instance, if you would like to query data for 07/01, you can do so from 07/03 to 07/24.
```
Comment (Danielabom) [inline concepts/data-connect-datasets.md:70]: The data available is from the last 21 days (about three weeks). For instance, if you would like to query data for 07/01, you can do so from 07/03 to 07/24.

File: concepts/data-connect-datasets.md [hunk 6]
```diff
@@ -14,26 +14,71 @@ Microsoft Graph Data Connect supports a variety of datasets, data regions, and s
...
+
+### OneDrive and SharePoint Online
+
+Please consider that these datasets are available after 48 hours (about 2 days). For instance, you can query data for 07/01 starting on 07/03.  
+
+This data is available for only the past 21 days (about 3 weeks). For instance, if you would like to query data for 07/01, you can do so from 07/03 to 07/24. 
+
+| Dataset name | Description | Sample and Schema | Billing |
+|--|--|--|--|
+| DocumentSharingDataset_v0_Preview | Contains information about sharing permissions of documents. | <ul><li> [Sample](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/sampledatasets/DocumentSharingDataset_v0_Preview.json)</li><li>[Schema](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/datasetschemas/DocumentSharingDataset_v0_Preview.md)</li></ul> | [Free (under preview only)](https://azure.microsoft.com/pricing/details/graph-data-connect/). These datasets will be available to those in Preview. You will need to join the Graph TAP program using the signup form at https://aka.ms/GraphTAPForm  and request to join the Preview.  |
```
Comment (Danielabom) [inline concepts/data-connect-datasets.md:74]:
```suggestion
| DocumentSharingDataset_v0_Preview | Contains information about sharing permissions of documents. | <ul><li> [Sample](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/sampledatasets/DocumentSharingDataset_v0_Preview.json)</li><li>[Schema](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/datasetschemas/DocumentSharingDataset_v0_Preview.md)</li></ul> | [Free (under preview only)](https://azure.microsoft.com/pricing/details/graph-data-connect/). These datasets will be available to those in Preview. You must join the Graph TAP program using the signup form at https://aka.ms/GraphTAPForm  and request to join the Preview.  |
```

Comment (Danielabom) [inline concepts/data-connect-datasets.md:74]:
```suggestion
| DocumentSharingDataset_v0_Preview | Contains information about sharing permissions of documents. | <ul><li> [Sample](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/sampledatasets/DocumentSharingDataset_v0_Preview.json)</li><li>[Schema](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/datasetschemas/DocumentSharingDataset_v0_Preview.md)</li></ul> | [Free (under preview only)](https://azure.microsoft.com/pricing/details/graph-data-connect/). These datasets will be available to those in Preview. You must join the Graph TAP program using the signup form at https://aka.ms/GraphTAPForm  and request to join the Preview.  |
```

File: concepts/data-connect-datasets.md [hunk 7]
```diff
@@ -14,26 +14,71 @@ Microsoft Graph Data Connect supports a variety of datasets, data regions, and s
...
+|VivaInsightsDataSet_v0  | Contains SharePoint group information, including details about group members.   | <ul><li> [Sample](https://github.com/niblak/dataconnect-solutions/blob/vivaarmtemplates/sampledatasets/VivaInsightsDataset_v0.json)</li><li>[Schema](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/datasetschemas/SharePointGroupsDataset_v0_Preview.md)</li></ul> | [Free (under preview only)](https://azure.microsoft.com/pricing/details/graph-data-connect/). These datasets will be available to those in Preview. You will need to join the Graph TAP program using the signup form at https://aka.ms/GraphTAPForm  and request to join the Preview. |
```
Comment (Danielabom) [inline concepts/data-connect-datasets.md:76]:
```suggestion
|VivaInsightsDataSet_v0  | Contains SharePoint group information, including details about group members.   | <ul><li> [Sample](https://github.com/niblak/dataconnect-solutions/blob/vivaarmtemplates/sampledatasets/VivaInsightsDataset_v0.json)</li><li>[Schema](https://github.com/microsoftgraph/dataconnect-solutions/blob/main/datasetschemas/SharePointGroupsDataset_v0_Preview.md)</li></ul> | [Free (under preview only)](https://azure.microsoft.com/pricing/details/graph-data-connect/). These datasets will be available to those in Preview. You must join the Graph TAP program using the signup form at https://aka.ms/GraphTAPForm  and request to join the Preview. |
```

File: concepts/data-connect-datasets.md (hunk 5 above)
Comment (David1997sb) [reply concepts/data-connect-datasets.md]: Changes applied.

**Comment by David1997sb** [comment]
@Danielabom please proceed with your review. All your feedback has been addressed.

File: concepts/data-connect-faq.md [hunk 8]
```diff
@@ -52,7 +52,7 @@ If you want to build an app for others to use to extract their Microsoft 365 dat
 
 ## Are service principals required with Data Connect?
 
-When creating the Data Factory pipeline, you have to provide a service principal to the Microsoft 365 linked service. In Azure, a service principal is a security identity that represents an application/service (as opposed to a user). Microsoft Graph Data Connect uses this service principal as its identity when getting authorized access to your Microsoft 365 data.
+When creating the Synapse or Data Factory pipeline, you have to provide a service principal to the Microsoft 365 linked service. In Azure, a service principal is a security identity that represents an application/service (as opposed to a user). Microsoft Graph Data Connect uses this service principal as its identity when getting authorized access to your Microsoft 365 data.
```
Comment (Danielabom) [inline concepts/data-connect-faq.md:55]: Please make sure you use the official product/service name. For example: Azure Synapse.

File: concepts/data-connect-pam.md [hunk 9]
```diff
@@ -147,7 +147,7 @@ Microsoft Graph Data Connect approval requests have particular characteristics t
 * Approval requests are based on the Azure Synapse or Azure Data Factory, pipeline, and copy activity names. Every copy activity run verifies that the Microsoft 365 admin has approved the copy activity's request to access Office data, and validates the important parameters of the copy activity run against the parameters of the approval.
 * Under certain conditions, a new approval request is automatically triggered. A Data Connect approver must approve the new request before the copy activity can access Microsoft 365 data.
 * If the parameters of the copy activity run changes, a new approval request is triggered.
-* If the Azure Synapse or Azure Data Factory, pipeline, or copy activity names change, a new approval request is triggered. For example, a new approval is required if the data table or set of columns that the copy activity is accessing changes.
+* If the Azure Synapse or Azure Data Factory pipeline or copy activity's name change, a new approval request is triggered. For example, a new approval is required if the data table or set of columns that the copy activity is accessing changes.
```
Comment (Danielabom) [inline concepts/data-connect-pam.md:150]: A new approval request is triggered if the Azure Synapse or Azure Data Factory pipeline or copy activity's name change. For example, a new approval is required if the data table or set of columns that the copy activity is accessing changes.

**Review by Danielabom — CHANGES_REQUESTED** [review]
Hi @David1997sb , thank you for working on this PR. Please review my comments.

File: concepts/data-connect-pam.md (hunk 9 above)
Comment (David1997sb) [reply concepts/data-connect-pam.md]: Done

File: concepts/data-connect-faq.md (hunk 8 above)
Comment (David1997sb) [reply concepts/data-connect-faq.md]: Done

**Comment by David1997sb** [comment]
Feedback addressed @Danielabom

File: concepts/data-connect-pam.md [hunk 10]
```diff
@@ -147,7 +147,7 @@ Microsoft Graph Data Connect approval requests have particular characteristics t
 * Approval requests are based on the Azure Synapse or Azure Data Factory, pipeline, and copy activity names. Every copy activity run verifies that the Microsoft 365 admin has approved the copy activity's request to access Office data, and validates the important parameters of the copy activity run against the parameters of the approval.
 * Under certain conditions, a new approval request is automatically triggered. A Data Connect approver must approve the new request before the copy activity can access Microsoft 365 data.
 * If the parameters of the copy activity run changes, a new approval request is triggered.
-* If the Azure Synapse or Azure Data Factory, pipeline, or copy activity names change, a new approval request is triggered. For example, a new approval is required if the data table or set of columns that the copy activity is accessing changes.
+* If the Azure Synapse or Azure Data Factory pipeline or copy activity's name change, a new approval request is triggered if the Azure Synapse or Azure Data Factory pipeline or copy activity's name change. For example, a new approval is required if the data table or set of columns that the copy activity is accessing changes.
```
Comment (Danielabom) [inline concepts/data-connect-pam.md:150]: Hi @David1997sb, my suggestion was: 
```suggestion
*  A new approval request is triggered if the Azure Synapse or Azure Data Factory pipeline or copy activity's name change. For example, a new approval is required if the data table or set of columns that the copy activity is accessing changes.
```

**Review by Danielabom — APPROVED** [review]
Changes look good. Thanks, @David1997sb

**Review by David1997sb — APPROVED** [review]

**Pull request merged**